src/Setup/WixBA/WixBA.cs

```diff
@@ -61,6 +63,65 @@ public static void LaunchUrl(string uri)
         }
 
         /// <summary>
+        /// Open a log file.
+        /// </summary>
+        /// <param name="uri">URI to a log file.</param>
+        internal static void OpenLog(Uri uri)
+        {
+            // Switch the wait cursor since shellexec can take a second or so.
+            System.Windows.Input.Cursor cursor = WixBA.View.Cursor;
```

Comment:
Not sure that I understand but I will take a look at it tomorrow.

On Thu, Jul 28, 2016 at 3:10 AM, Rob Mensching <notifications@github.com>
wrote:

> In src/Setup/WixBA/WixBA.cs
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72580457>:
>
> > @@ -61,6 +63,65 @@ public static void LaunchUrl(string uri)
> >          }
> >
> >          /// <summary>
> > +        /// Open a log file.
> > +        /// </summary>
> > +        /// <param name="uri">URI to a log file.</param>
> > +        internal static void OpenLog(Uri uri)
> > +        {
> > +            // Switch the wait cursor since shellexec can take a second or so.
> > +            System.Windows.Input.Cursor cursor = WixBA.View.Cursor;
>
> The bulk of this should be refactored to be a reusable
> ShellExecuteWithBetterCursorAndBetterNameThanRobCameUpWithHere() method.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/5b08dba511d7438508767effab67896442e4ad1b#r72580457>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLVcwB-JRSi8RMbQ6RccYgnq50ISdks5qaGQNgaJpZM4JVm8P>
> .
>